content/workshops/node/_index.md

```diff
@@ -1,11 +1,18 @@
 ---
-title: Node 
+title: Node
+ready: true
 ---
 
-## Introduction (slides)
+## Introduction
 
-- [Introduction to Node](https://docs.google.com/presentation/d/1dtrsWOWjoA9A9nTmznCkdYz0oRK6O76jXRINp0tzjrQ/edit?usp=sharing)
+The followings are to be discussed, demonstrated and addressed during the workshop
 
-- [Asynchronous vs Synchronous](https://docs.google.com/presentation/d/19ppsjmkKAvJcasK_BnjptTUfF4KQwaInDKVkhK1b6vA/edit?usp=sharing)
+1. What is Node.js
+2. Why Node js
+3. Node installation process
+4. NPM vs NPM
+5. Asynchronous vs Synchronous
 
-## Challenges 
\ No newline at end of file
+## Resources
+
```

Comment:
Do we have a node topic that can be updated with these resources? Ideally all the `topic` like content will be inside a topic. 

Documentation should be like code: DRY and cohesive. This is not cohesive.

Reply:
No topic exists yet for node.js at the moment.

Reply:
Then make a topic and put this link in it.

People might want to search for node topics within the repo. If we have resources then they should be easy to find.